dotnet/src/dotnetframework/GxClasses/Model/GXWebProcedure.cs

```diff
@@ -100,6 +107,19 @@ protected override void sendCacheHeaders()
 				}
 			}
 		}
+
+		private void setOuputFileName()
+		{
+			if (!string.IsNullOrEmpty(outputFileName))
+			{
+				context.HttpContext.Response.AddHeader(HttpHeader.CONTENT_DISPOSITION, "inline; filename=" + outputFileName + ".pdf");
```

Comment:
Should we take into account the Extension based on the "output_file rule (or maybe with the "getOutputType()")"?

`Output_file('TicketReservation','PDF');`

It seems that the second parameter (format) allows other values:

> format
>       Is the output file format. Possible values are ‘GXR’, ‘RTF’, ‘XML’ ‘PDF’ or ‘TXT’ (the last one only for reports in text mode). It is ignored in the case of output through screen or printer.

https://wiki.genexus.com/commwiki/servlet/wiki?7965,Output_File%20rule

Reply:
Agree. Fixed now. It must take into account the type.